docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# How to Contribute to Playnite Web
+
+[![Build Status](https://github.com/andrew-codes/playnite-web/actions/workflows/verify-commits.yml/badge.svg)](https://github.com/andrew-codes/playnite-web/actions/workflows/verify-commits.yml)
+[![Latest Release](https://img.shields.io/github/v/release/andrew-codes/playnite-web)](https://github.com/andrew-codes/playnite-web/releases/latest)
+[![License](https://img.shields.io/github/license/andrew-codes/playnite-web)](https://github.com/andrew-codes/playnite-web?tab=AGPL-3.0-1-ov-file#readme)
+[![GitHub Sponsors](https://img.shields.io/github/sponsors/andrew-codes)](https://github.com/sponsors/andrew-codes)
+[![Discord](https://img.shields.io/discord/1432357177984684102?style=flat&label=Discord%20Community)](https://discord.gg/Dv8UrQ6mGb)
+
+- [Report a Bug](#report-a-bug)
+- [Contribute a Bug Fix](#contribute-a-bug-fix)
+- [Add or Update a Feature](#add-or-update-a-feature)
+- [Contribute to the Documentation](#contribute-to-the-documentation)
+- [Learn More About the Source Code](#learn-more-about-the-source-code)
+- [Thank You](#thank-you)
+
+## Report a Bug
+
+> **Note:** If you discover a security vulnerability, **do not open an issue**. To report a security bug, [submit a security advisory on GitHub](https://github.com/andrew-codes/playnite-web/security/advisories/new).
+
+1. **Make sure you're reporting a new bug** by searching any currently [open issues](https://github.com/andrew-codes/playnite-web/issues) on GitHub.
+2. If you don't find an open issue about the bug [open a new one](https://github.com/andrew-codes/playnite-web/issues/new). 
+3. Include a clear title and description, as much relevant information as possible, and any relevant screenshots that show the problem.
+
+## Contribute a Bug Fix
+
+1. Read the [Playnite Web Conventions](contributor-guides/conventions-guide.md) guide to learn more about coding conventions and general guidelines.
+2. Open a new GitHub pull request with the patch.
+3. Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+## Add or Update a Feature
```

Comment:
Maybe we mention the `feat: ` prefix. I'm not certain if there should be a central section for this or underneath each respective type of contribution. Open to suggestions.